stable/admin/files/nuoadmin

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+. ${NUODB_HOME}/etc/nuodb_setup.sh
+
+if [ -e ${NUODB_CFGDIR}/jvm-options -a "$NUODB_JAVA_OPTS" = "" ] ; then
+    . ${NUODB_CFGDIR}/jvm-options
+fi
+
+: ${NUODB_SERVERID:=$(hostname)}
+: ${NUODB_ALT_ADDRESS:=$(hostname --fqdn)}
+: ${NUODB_ADMIN_PORT:="48005"}
+
+if [ "${NUODB_ALTERNATIVE_ENTRYPOINT}" != "" ] ; then
+    if nuodocker --api-server "${NUODB_ALTERNATIVE_ENTRYPOINT}:8888" check servers ; then
+        # Admin server exists at alternative entrypoint. Use it instead
+        NUODB_DOMAIN_ENTRYPOINT="${NUODB_ALTERNATIVE_ENTRYPOINT}"
+    fi
+fi
+
+# if endpoint is not specified, default to bootstrap server
+if [ "${NUODB_DOMAIN_ENTRYPOINT}" = "" ] ; then
+    NUODB_DOMAIN_ENTRYPOINT="${NUODB_BOOTSTRAP_SERVERID}"
+fi
+
+export NUODB_LOGDIR NUODB_CFGDIR NUODB_VARDIR NUODB_BINDIR NUODB_RUNDIR
+
+# attempt to retain the previous crash directory (within the configured window to avoid filling the disk)
+if [ -f $NUODB_CRASHDIR/* ]; then
+  crashcount=$(find $NUODB_LOGDIR/crash-* -type d -cmin -$OVERWRITE_WINDOW ! -path . | wc -l)
```

Comment:
@acabrele Have you got an example of how the log directory content will look like after engine crash multiple times?
I thought that the core files are located in `$NUODB_LOGDIR/crash/core.nuodb.*` or `$NUODB_CRASHDIR`. Did you mention something about the core file overwriting because of the same name?

Reply:
If logPersistence is not enabled this log-volume will be created as it used to be, so the core would still go there, just still be lost on crashloop.

So the overwrite happens inside $NUODB_LOGDIR/crash/.  This moves any contents that exist to $NUODB_LOGDIR/crash-_date_/ allowing a new core to create in $NUODB_LOGDIR/crash/ without overwriting if the process ID was the same.  I've noticed in a crash loop the PID does change.  The issue would occur when someone deletes or scales down/up the container in an attempt to fix an issue.  In that case the re-start is likely to used the same PID as the original crash and then overwrite the interesting one.

Reply:
For ReadWriteMany, I needed to prioritise the contrail stuff so will come back to that.  Hoping the NuoDB solution will be along soon to address it too.  Ultimately if it can't support GCP it is not useful for the immediate requirement.

Reply:
... I thought we used to put a date-time stamp on core file names.
- Is that no longer the case?

Reply:
This is the example from WeLab
/var/log/nuodb/crash/core.nuodb.sm-database-wlab-aws0-t24-1.63